projects/kit/components/select/native-select/native-select.component.ts

```diff
@@ -0,0 +1,39 @@
+import {ChangeDetectionStrategy, Component, Input, TemplateRef} from '@angular/core';
+import {TuiDataListDirective} from '@taiga-ui/core';
+
+import {AbstractTuiNativeSelect} from './native-select';
+
+@Component({
+    selector: `select[tuiSelect]:not([labels])`,
+    templateUrl: `./native-select.template.html`,
+    providers: [
+        {
+            provide: TuiDataListDirective,
+            deps: [TuiNativeSelectComponent],
+            useExisting: TuiNativeSelectComponent,
+        },
+        {
+            provide: TemplateRef,
+            deps: [TuiNativeSelectComponent],
+            useFactory: ({datalist}: TuiNativeSelectComponent) => datalist,
+        },
+        {
+            provide: AbstractTuiNativeSelect,
+            useExisting: TuiNativeSelectComponent,
+        },
+    ],
+    host: {
+        '[attr.aria-invalid]': `host.invalid`,
+        '[attr.disabled]': `host.disabled || null`,
```

Comment:
It can be `'[disabled]': 'host.disabled'`. We use `attr.disabled` for buttons because they can also be on `a` tag which doesn't have that property.